File: mult.py
#!/usr/bin/env python

# Mikaels multiplikationstabell

def multiplication(rows, cols):
    out = ""

    for i in range(1, rows + 1):
        for j in range(1, cols + 1):
            out += (str(i*j) + '\t')

        out += '\n'

    return out

if __name__ == '__main__':
    print(multiplication(input("How many rows? "), input("Hur many columns? ")))

File: mult.cpp
// Mikaels multiplikation

#include <iostream>
#include <string>

using namespace std;

void multiplication(int rows, int cols) {;
  for(int i=1; i <= rows; i++) {
    for(int j=1; j <= rows; j++) {
      cout << i*j << "\t";
    }

    cout << endl;

  }
}

int main() {
  int rows, cols;
  //int cols;

  cout << "How many rows? ";
  cin >> rows;

  cout << "How many columns? ";
  cin >> cols;

  multiplication(rows, cols);

  return 0;
}
